### PR TITLE
Add config for printer Creality CR10s Pro V2

### PR DESCRIPTION
**Pull request by flatline-84**
Created a base config for the Creality CR10s Pro v2 as described here: https://www.creality.com/products/cr-10s-pro-v2-3d-printer

This will allow users with this printer to more easily setup Klipper firmware.

### CODE REVIEW

**Comment by JamesH1978** [comment]
Thankyou for submitting a PR, please could you refer to point 3 in the "what to expect" section in https://github.com/Klipper3d/klipper/blob/master/docs/CONTRIBUTING.md and add a signed-off-by line and point 4 in https://github.com/Klipper3d/klipper/blob/master/docs/Example_Configs.md and include the regression tests.

Thanks
James

**Comment by flatline-84** [comment]
Hi James, I have addressed all the points. Hope this is correct

File: config/printer-creality-cr10s-pro-v2-2020.cfg [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+serial: /dev/serial/by-id/<YOUR_USB_ID>
+
+[printer]
+kinematics: cartesian
+max_velocity: 200
+max_accel: 1500
+max_z_velocity: 10
+max_z_accel: 100
+
+[gcode_macro G29]
```
Comment (JamesH1978) [inline config/printer-creality-cr10s-pro-v2-2020.cfg:17]: Please do not include macros in a config that do not pertain to base functionality of a printer, BED_MESH_CALIBRATE is perfectly valid gcode in its own right.

Thanks
James

File: config/printer-creality-cr10s-pro-v2-2020.cfg [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+[filament_switch_sensor e0_sensor]
+switch_pin: PE4 #ar2
+pause_on_runout: False
+runout_gcode:
+  PAUSE_PARK
+
+## Calibrating the Bed
+# this is for the bltouch to calibrate the bed
+[screws_tilt_adjust]
```
Comment (JamesH1978) [inline config/printer-creality-cr10s-pro-v2-2020.cfg:134]: You have the same screw positions for this and bed_screws, this is not likely right, as one needs absolute coordinates when probe is over the screw(screws_tilt_adjust), and the other needs absolute coordinates when nozzle is over screw(bed_screws).

Thanks
James

File: config/printer-creality-cr10s-pro-v2-2020.cfg [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+screw2_name: front right screw
+screw3: 295,280
+screw3_name: rear right screw
+screw4: 40,280
+screw4_name: rear left screw
+speed: 50
+horizontal_move_z: 10
+screw_thread: CW-M3
+
+[bed_screws]
```
Comment (JamesH1978) [inline config/printer-creality-cr10s-pro-v2-2020.cfg:147]: see above comment

**Review by JamesH1978 — CHANGES_REQUESTED** [review]
Thank you for making the changes i asked, please see my comments.

Thanks
James

File: config/printer-creality-cr10s-pro-v2-2020.cfg (hunk 1 above)
Comment (flatline-84) [reply config/printer-creality-cr10s-pro-v2-2020.cfg]: Done

File: config/printer-creality-cr10s-pro-v2-2020.cfg (hunk 2 above)
Comment (flatline-84) [reply config/printer-creality-cr10s-pro-v2-2020.cfg:130]: Fixed

**Review by JamesH1978 — APPROVED** [review]
Looks fine to me.

Thanks
James

**Comment by KevinOConnor** [comment]
Thanks.  Could you also update the pid lines to use `option: value` syntax instead of `option = value` syntax?  https://www.klipper3d.org/Example_Configs.html point 7d.

-Kevin

**Comment by flatline-84** [comment]
Good catch Kevin. Thanks, updated now

**Review by JamesH1978 — APPROVED** [review]
Apologies for missing the syntax,

Still looks good

Thanks
James

**Comment by KevinOConnor** [comment]
Alas, the `option = value` is still present in the heater_bed section.

Cheers,
-Kevin

**Comment by flatline-84** [comment]
I could have sworn I did a find + replace but apparently not. Should be properly good to go now

**Comment by KevinOConnor** [comment]
Thanks.

-Kevin

**Pull request merged**